src/elements/basic/ButtonElement.jsx

```diff
@@ -36,6 +36,23 @@ function applyButtonStyles(element, applyStyles) {
   applyStyles.applyWidth('button');
   applyStyles.applyCorners('button');
   applyStyles.applyBorders('button');
+  applyStyles.applyFlexDirection('button');
+  applyStyles.apply(
+    'button',
+    [
+      'uploader_padding_top',
+      'uploader_padding_right',
+      'uploader_padding_bottom',
+      'uploader_padding_left'
+    ],
+    (a, b, c, d) => ({
+      padding: `${a}px ${b}px ${c}px ${d}px`
+    })
+  );
+  // not really sure if these should be true
```

Comment:
maybe not since we probably want image aspect ratio respected over the height and width values?